content/learn/book/welcome/_index.md

```diff
@@ -6,4 +6,44 @@ template = "book-section.html"
 page_template = "book-section.html"
 +++
 
-TODO: Briefly discuss what Bevy is.
+Welcome to the Bevy game engine! A bevy may be a group of birds, but our engine is:
+
+- Rust-first: with all the safety, tooling and power that comes with a modern systems language
+- completely free, open source and hackable with an architecture designed for modularity
+- powered by an ergonomic and innovative ECS that makes writing clear, performant gameplay and engine logic a joy
+- cross-platform: support all major desktop platforms with a single code-base, with limited mobile and web support
```

Comment:
I think each bullet should take the form '**Snappy Thing**: longer single-sentence description". I think that flows better and helps the user internalize each point (see the section that this replaces in the current book).